src/Components/WishListProduct.js

```diff
@@ -1,157 +1,72 @@
-                alt="product"
-              />
-            </div>
-          </div>
-          <div className="card__secondary text-center card__secondary-vertical">
-            <p className="p1">Men Premium Jacket</p>
-            <h2 className="card__subtitle text-center">Rs. 2000</h2>
-          </div>
-          <div className="card__action-buttons text-center">
-            <button className="button button-primary p1 btn-full btn-wishlist bolder">
-              Go To Cart
-            </button>
-          </div>
-        </div>
-
-        <div className="card-vertical">
-          <div className="card__primary-action card__primary-action-column card__primary-action-vertical">
-            <span className="material-icons-outlined badge-up-right-corner card-badge card-badge-vertical card-badge-check">
-              <FavoriteIcon />
-            </span>
-            <div className="card__media-column card__media-column-vertical">
-              <img
-                className="card-img"
-                src="https://raw.githubusercontent.com/Omkar-Chaskar/e-commerce-app/dev/image/product-1.png"
-                alt="product"
-              />
-            </div>
-          </div>
-          <div className="card__secondary text-center card__secondary-vertical">
-            <p className="p1">Men Premium Jacket</p>
-            <h2 className="card__subtitle text-center">Rs. 2000</h2>
-          </div>
-          <div className="card__action-buttons text-center">
-            <button className="button button-primary p1 btn-full btn-wishlist bolder">
-              Go To Cart
-            </button>
-          </div>
-        </div>
-      </div>
+      {wishlistState.wishlist < 1 ? (
+        <p className="p3 bold center">
+          There is no item in wishlist. Please add some items in wishlist.
+        </p>
+      ) : (
+        <ul className="product-container">
+          {wishlistState.wishlist &&
+            wishlistState.wishlist.map((products) => {
+              return (
+                <li className="card-vertical" key={products._id}>
+              <div className="card__primary-action card__primary-action-column card__primary-action-vertical">
+                <span className="material-icons-outlined badge-up-right-corner card-badge card-badge-vertical">
+                      <FavoriteIcon
+                        onClick={() =>
+                          wishlistDispatch({
+                            type: "REMOVE_FROM_WISHLIST",
+                            payload: products
+                          })
+                        }
+                      />
+                    </span>
+                <div className="card__media-column card__media-column-vertical">
+                  <img
+                    className="card-img-height"
+                    src={products.productImg}
+                    alt=""
+                  />
```

Comment:
use prettier formatter to remove these extra spaces

Reply:
thanks